ravem/indico_ravem/client/index.js

```diff
@@ -192,7 +195,7 @@
     /**
      *  Sets a new state for the button and update its icon, label and tool tip.
      */
-    function setButtonState(btn, newState, tooltipMessage) {
+    function setButtonState(btn, newState, tooltipMessage = '', currentVCRoom = '') {
```

Comment:
Does `tooltipMessage` need a default or can it be without?

Reply:
it could be without, just feels nicer since we were not passing one before so it was just undefined anyway

Reply:
yeah I don't have a strong opinion on it..